src/lib/es5.d.ts

```diff
@@ -1597,7 +1549,7 @@ interface Int8Array {
       */
     every(callbackfn: (this: void, value: number, index: number, array: Int8Array) => boolean): boolean;
     every(callbackfn: (this: void, value: number, index: number, array: Int8Array) => boolean, thisArg: undefined): boolean;
-    every<Z>(callbackfn: (this: Z, value: number, index: number, array: Int8Array) => boolean, thisArg: Z): boolean;
+    every<Z>(callbackfn: ( value: number, index: number, array: Int8Array) => boolean, thisArg: Z): boolean;

```

Comment:
Something isn't right here. All of the native array types still have three overloads per method, all that changed was that the `this` parameter disappeared from the last overload (and got replaced with a blank).